tests/helper/component_interface.go

```diff
@@ -11,8 +11,11 @@ type Component interface {
 	ExpectIsDeployed()
 	// ExpectIsNotDeployed checks that the component is not deployed
 	ExpectIsNotDeployed()
-	// Exec executes the command in specific container of the component
-	Exec(container string, args ...string) string
+	// Exec executes the command in specific container of the component.
+	// If success is true, the command exit code is expected to be 0.
+	// If success is false, the command exit code is expected to be non-zero.
+	// If success is nil, the command is just supposed to run, with no assertion on its exit code.
+	Exec(container string, success *bool, args ...string) (string, string)
```

Comment:
I think that output parameters are generally placed at the end of the parameters list, after input parameters.

Could you move the `success` param at the end (and rename it `expectedSuccess` to make its meaning clearer)?

Reply:
Done - see [`9e3f352` (#6543)](https://github.com/redhat-developer/odo/pull/6543/commits/9e3f3525e0f0b7bfb402492bce852dda0499b5cf)